### ISSUE TITLE
reproduction problem

### ISSUE THREAD

**Issue by zhao2000hhh** [NiFangBaAGe/CoordFill#6]
When I retrained with your training code and configuration, I couldn't produce a model as good as yours。
Specifically, I tried on the celeba-HQ dataset.
mask ftom  https://nv-adlr.github.io/publication/partialconv-inpainting
train on 20-30% subset
test on the whole mask set
my model：
psnr=28.5719, val: ssim=0.8786, val: lpips=0.0945
your model：
PSNR = 29.581334   SSIM= 0.893  LPIPS=0.08094
Do you have any special training techniques？
Looking forward to your reply.

**Comment by NiFangBaAGe** [comment]
Thanks for your interest in our work.
The masks for experiments are generated from https://nv-adlr.github.io/publication/partialconv-inpainting. We dilate or erode them to ensure the mask ratio is not more than 25% (following HiFill).

**Comment by zhao2000hhh** [comment]
Is the testing also conducted on the 25%mask？

**Comment by NiFangBaAGe** [comment]
Yes, all the training and testing masks follow this setting.

**Comment by zhao2000hhh** [comment]
thank you reply. By the way, could you share me the 25% mask?

**Comment by zhao2000hhh** [comment]
looking forward to your reply. sos

**Comment by NiFangBaAGe** [comment]
The mask is available [here](https://uofmacau-my.sharepoint.com/:f:/g/personal/mc05379_umac_mo/EhBo9KI9uQRKtHd-3QGxHKABNzj8K8NG1Z6J5_YmyjoYFw?e=Y75flT). Hope it can help you.

**Comment by zhao2000hhh** [comment]
good lunck

**Comment by zhouwer** [comment]
> When I retrained with your training code and configuration, I couldn't produce a model as good as yours。 Specifically, I tried on the celeba-HQ dataset. mask ftom https://nv-adlr.github.io/publication/partialconv-inpainting train on 20-30% subset test on the whole mask set my model： psnr=28.5719, val: ssim=0.8786, val: lpips=0.0945 your model： PSNR = 29.581334 SSIM= 0.893 LPIPS=0.08094 Do you have any special training techniques？ Looking forward to your reply.

Hi,recently, I have also been preparing to train this model, but I found that when building the training set, only two sets of images are inputted, root_path1 and root_path2, rootpath2 is mask ,So, what is root_path1? i think need three images as input, one is image with object,two is image without object ,three is mask of object ,right? but why this model only two image?   root_path1 is  image with object  or  is image without object ? thinks!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
